Cargo.toml

```diff
@@ -31,3 +31,8 @@ atty = "0.2"
 matrix-sdk = { version = "0.4", default-features = false, features = ["rustls-tls", "markdown"] }
 mime = "0.3"
 mime_guess = "2.0"
+
+[profile.release]
+strip = "symbols"
+opt-level = "s"
```

Comment:
I am fine with trading compile time for binary size.
But i don't like the idea of trading run time performance for binary size.
My testing showed that `opt-level = "s"` only makes a small difference `7.8M` vs `6.7M`.

But if you want the additional `1.1M` savings that `opt-level = "s"` provides it could be added as a custom profile:
```toml
[profile.release-tiny]
inherits = "release"
opt-level = "s"
```

Reply:
Thanks for your feedback on this!

> But i don't like the idea of trading run time performance for binary size.

Seems reasonable, I haven't tested performance metrics in that particular case, but having a performance malus to spare 1.1mb seems unreasonable. I think that having a release profile only containing:

```
[profile.release]
strip = "symbols"
lto = true
```

and adding, as you have suggested, an additional release-tiny profile:

```
[profile.release-tiny]
inherits = "release"
opt-level = "s"
```

should work well. I've included these changes to this PR.